CHANGES.md

```diff
@@ -21,8 +21,106 @@ this file to understand what changed.
+    - Endpoint Interruption Logic for Model Overview with QA by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2274
+    - add e2e UI notebook tests to blbooksgenre text classification notebook by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2273
+  - ## RAI Vision
+    - Require pillow>=10.0.0, catch drise failure due to low mask_res or num_masks by @romanlutz in https://github.com/microsoft/responsible-ai-toolbox/pull/2181
+    - Add retry logic to image reader by @kicha0 in https://github.com/microsoft/responsible-ai-toolbox/pull/2206
+    - fix error for automl object detection models when initializing error analysis and optimize explanation execution by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2245
+    - fix for aggregate parameter passed to torchmetrics by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2235
+    - pass transforms for optimized automl model explanations and improve error logging to include inner exception by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2269
+  - ## RAI Text
+    - add RAITextInsights DBPedia notebook example and tests by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2204
+    - add RAITextInsights covid19 multilabel classification notebook example and tests by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2221
+    - add RAITextInsights blbooksgenre binary classification notebook example and tests by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2226
+  - ## RAI Tabular
+    - Change `ValueError` to `UserConfigValidationException` in casual_manager.py by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2209
+    - Add input type validation for FeatureMetadata class by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2218
+    - Move missing value validations before unique value validations by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2228
+    - Pass categorical features after dropping dropped features in CounterfactualManager by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2241
+    - RAIInsights: prevent failures on optional methods & fix feature range formatting for timestamps by @romanlutz in https://github.com/microsoft/responsible-ai-toolbox/pull/2258
+    - fix RAI explanation error for dropped categorical features after serialization by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2261
+    - Tabular Only: Address missing values as per each manager in responsibleai by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2263
+    - responsibleai: Add prediction wrapper to missing value tests by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2270
+  - ## error analysis
```

Comment:
nit: Error analysis in sentence case

Reply:
nvm it seems it should be caps for EA going along with previous docs:
Error Analysis